atomicapp/utils.py

```diff
@@ -377,6 +377,53 @@ def rm_dir(directory):
         distutils.dir_util.remove_tree(directory)
 
     @staticmethod
+    def getUid(user):
+        """
+        Get the UID of the specific user by grepping /etc/passwd unless
+        we are in a container.
+
+        Returns:
+            (int): User UID
+        """
+
+        # If we're in a container we should be looking in the /host/ directory
+        if Utils.inContainer():
+            passwd = os.path.join(HOST_DIR, "etc/passwd")
+        else:
+            passwd = "/etc/passwd"
```

Comment:
The problem I found is getting the UID from within the container (hence why
I do it via /etc/passwd). How could we possibly do this from within the
container?